ApplicationLibCode/ProjectDataModel/StimPlanModel/RimStimPlanModelPressureCalculator.h

```diff
@@ -57,4 +57,10 @@ class RimStimPlanModelPressureCalculator : public RimStimPlanModelWellLogCalcula
                                        std::vector<double>&      values,
                                        std::vector<double>&      measuredDepthValues,
                                        std::vector<double>&      tvDepthValues ) const;
+
+    bool interpolateInitialPressureByEquilibrationRegion( const RimStimPlanModel*    stimPlanModel,
```

Comment:
I think the most common phrase to use is EquilibriumRegion, not EquilibrationRegion